supporting_files/AtlasSiteConfig.yml

```diff
@@ -200,7 +200,7 @@ min_pval_dir: /nfs/production3/ma/home/atlas3-production/minPValGt0.5
 
 failed_curation_dir: /nfs/production3/ma/home/atlas3-production/failedCuration
 
-non_standard_experiments_file: <CONDA_PREFIX>/atlasprod/supporting_files/non_standard_experiments.yml
+non_standard_experiments_file: <CONDA_PREFIX>/atlasprod/perl-atlas-modules/supporting_files/non_standard_experiments.yml
```

Comment:
I wonder if this path changes could have been more easily resolved in atlas prod by a symlink instead, where `supporting_files` would point to `perl-atlas-modules/supporting_files` and then we avoid all these internal path changes.

Reply:
This is possible when working merged dev/master in atlas-prod having fixed link. For testing feature branches this may not work with changing symlinks.

Reply:
why would that work in one case and not in the other? I mean this on the atlas-prod repo.

Reply:
what I basically mean is that once the submodule is in place in atlas-prod, we go to the base directory and replace that `supporting_files` directory there with a link to `perl-atlas-modules/supporting_files`.  Then no script needs to be aware of any new path, because it is the same. We would only need to make sure that the bamboo process first does the submodule and then copies the files inside `supporting_files`.

Reply:
Ok, that sounds reasonable. Let me test this.